www/scipylib/building/linux.rst

```diff
@@ -378,11 +380,11 @@ ATLAS
 
 **There is currently no comfortable way to use ATLAS on openSuse.**
 
-The  `build instructions for ATLAS <http://www.scipy.org/Installing_SciPy/Linux#head-89e1f6afaa3314d98a22c79b063cceee2cc6313c>`__  on this page work, but unfortunately the NumPy and SciPy packages don't work with ATLAS. One could build NumPy and SciPy from sources though, and a relatively painless way to do this is the Sage package. (If you know a comfortable way to make ATLAS work on openSuse, please put it here into the Wiki.)
+The  `build instructions for ATLAS <http://www.scipy.org/install.html#head-89e1f6afaa3314d98a22c79b063cceee2cc6313c>`__  on this page work, but unfortunately the NumPy and SciPy packages don't work with ATLAS. One could build NumPy and SciPy from sources though, and a relatively painless way to do this is the Sage package. (If you know a comfortable way to make ATLAS work on openSuse, please put it here into the Wiki.)
```

Comment:
This is a link to a different section on this page, can probably be written as (implicit title target)
```
`build instructions for ATLAS <Building ATLAS>`_
```